### ISSUE TITLE
Start measuring and tracking Clubs application form success

### ISSUE THREAD

**Issue by hannahkane** [mozilla/teach.mozilla.org#1543]
We've made a few changes to the Clubs application form and workflow so far, and have a few more possible changes in mind. As part of this work, I'd like to start tracking the effects our changes. Specifically, I'd like to track:
 * the ratio of denied applicants to the total number of applicants
 * the reasons we deny applications
 * % of people who exit the process before completing the survey step

Goals:
 * improve quality of applications (i.e. reduce % of applications that get denied)
 * increase % of people who complete the process, without reducing the quality of applications

Process:
 * @kgorr - since you're in charge of reviewing Club submissions, you're probably in the best position to do track this. We could track this manually, but if that's too much of a burden, we could put some effort into tracking this more dynamically. For example, we could add a "Reason for Denial" field to the Clubs database. You could tell us what the common reasons are, so we could populate a pull-down menu, and you could choose each time you Deny an application.

Thoughts?  @kgorr @ldecoursy @amirad @adamlofting @mmmavis @Pomax

**Comment by adamlofting** [comment]
:+1: sounds sensible to me

**Comment by KGorr** [comment]
Having a reason for denial field would be great, that way anyone on the team can see it. The most common reasons are (in order of frequency):
- no response after initial application (they don't fill out interview questions)
- email bounces
- They aren't actually interested in being a Club Captain, just want info
- Not qualified
- Misunderstand what Mozilla Clubs are and what being a Club Captain means and aren't interested after further probing

**Comment by amirad** [comment]
Should we expand on what not qualified means? Aren't able to mentor others? Don't have a group to teach?

**Comment by KGorr** [comment]
Could be interested in swag (more than teaching the web); wrong motives, no connections.

Those are the reasons I've denied a club this far. Could be other reasons in the future...

**Comment by hannahkane** [comment]
This is really useful information, thanks.

Wondering what we do for those who just want info. Do we have a phone call with them? Do we point them to a resource? Maybe we can reduce that one by providing better navigation on the page itself.

**Comment by KGorr** [comment]
Good questions, @hannahkane. Also wondering if having a Q&A or link to a resource explaining the difference between a Firefox Club and Mozilla Club would help. I've received several of applications that seem to be more interested in browser related topics than web literacy as a whole. Thoughts @amirad?

**Comment by hannahkane** [comment]
Adding "blocked" because we can't start until we finalize what this ticket is. :)

**Comment by amirad** [comment]
@hannahkane can we point those people to the monthly office hours? http://mozilla.github.io/learning-networks/clubs/#officehours

**Comment by hannahkane** [comment]
Great ideas - the bit about differentiating from Firefox Clubs and adding the office hours. Will file a separate ticket for those.

**Comment by hannahkane** [comment]
Closing this ticket. Will file separate tickets based on discussion here.
